apps/meteor/client/views/admin/rooms/RoomsTable.tsx

```diff
@@ -121,7 +121,7 @@ const RoomsTable = ({ reload }: { reload: MutableRefObject<() => void> }): React
 				<GenericTable>
 					<GenericTableHeader>{headers}</GenericTableHeader>
 					<GenericTableBody>
-						<GenericTableLoadingTable headerCells={mediaQuery ? 6 : 3} />
+						<GenericTableLoadingTable data-id='RoomGenericTableLoadingTable' headerCells={mediaQuery ? 6 : 3} />
```

Comment:
we should not add `data-id` or similar things for test purposes only.

Reply:
This is a bad practice for tests.
https://playwright.dev/docs/best-practices#best-practices
https://testing-library.com/docs/queries/about/#priority

Basically you should always emulate how a user would select elements on a page, or maybe a screen reader. By adding these ids you're directly selecting the element that could not be accessible otherwise.